### PR TITLE
[Databricks] Implement sweeping volumes

### PR DESCRIPTION
**Pull request by Tang8330**
Volumes are where we are storing the delta files in Databricks, similar to temporary tables, we should purge them regularly. This PR adds that capability.

### CODE REVIEW

File: lib/sql/rows.go [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+func RowsToObjects(rows *sql.Rows) ([]map[string]any, error) {
+	defer rows.Close()
+
+	columns, err := rows.Columns()
+	if err != nil {
+		return nil, err
+	}
+
+	var objects []map[string]any
+	for rows.Next() {
```
Comment (nathan-artie) [inline lib/sql/rows.go:14]: You'll need to check if `rows.Err()` is not nil after iterating through `rows.Next()`.

Comment (nathan-artie) [reply lib/sql/rows.go:14]: https://github.com/golang/go/blob/356ba0f06586a833cd8de9c04af0d2adddf95851/src/database/sql/sql.go#L3122

Comment (Tang8330) [reply lib/sql/rows.go:17]: sg

File: lib/sql/rows.go [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+		row := make([]any, len(columns))
+		rowPointers := make([]interface{}, len(columns))
```
Comment (nathan-artie) [inline lib/sql/rows.go:19]: `any`.

**Review by nathan-artie — APPROVED** [review]

**Pull request merged**